app/scripts/components/exploration/components/map/share-button.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useCallback } from 'react';
+import { Icon } from '@trussworks/react-uswds';
+
+import { SelectorButton } from '$components/common/map/style/button';
+import useThemedControl from '$components/common/map/controls/hooks/use-themed-control';
+
+export function ShareButtonComponent({
+  onClick,
+  disabled
+}: {
+  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  disabled: boolean;
+}) {
+  return (
+    <SelectorButton
+      tipContent='Share Exploration'
+      tipProps={{ placement: 'left' }}
+      disabled={disabled}
+      onClick={onClick}
+    >
+      <Icon.Share size={3} />
+    </SelectorButton>
+  );
+}
+
+export function ShareButton({
+  setLinkCopied
+}: {
+  setLinkCopied: (copied: boolean) => void;
+}) {
+  const handleShare = useCallback(async () => {
+    try {
+      // Get current URL
+      const currentUrl = window.location.href;
+
+      // Call the URL shortening API
+      const response = await fetch(
+        `https://openveda.cloud/service/link/shorten?url=${encodeURIComponent(
```

Comment:
i think it's a good idea of setup this url as an env var instead of hardcoding it here